client/post-editor/editor-location/map.jsx

```diff
@@ -0,0 +1,109 @@
+/** @format */
+/* global google */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import qs from 'querystring';
+
+export default class extends React.Component {
+	static displayName = 'EditorLocationMap';
+
+	static propTypes = {
+		coordinates: PropTypes.array,
+		onError: PropTypes.func,
+		onSelect: PropTypes.func,
+	};
+
+	static defaultProps = {
+		coordinates: [],
+		onError: () => {},
+		onSelect: () => {},
+	};
+
+	componentDidMount() {
+		// Connect the initMap() function within this class to the global window
+		// context, so Google Maps can invoke it.
+		window.initMap = this.initMap.bind( this );
+
+		const src =
+			'https://maps.googleapis.com/maps/api/js?' +
+			qs.stringify( {
+				key: 'INSERT-API-KEY-HERE',
```

Comment:
Does the loading of the request fail without the `key` param?

Reply:
Yeah, it looks like this:

<img width="215" alt="screenshot 2017-11-27 22 43 52" src="https://user-images.githubusercontent.com/136891/33301592-8469f784-d3c4-11e7-9d3e-f581710b7f41.png">

I'm thinking it'd be good to fall back on the old map if there's no API key. I'll go ahead and implement that.